pytorchfi/core.py

```diff
@@ -362,25 +359,23 @@ def _set_value(self, module, input, output):
             for i in inj_list:
                 self.assert_inj_bounds(index=i)
                 logging.info(
-                    "Original value at [%d][%d][%d][%d]: %d"
```

Comment:
small fix: can you change the last %d to %f (since the value can be a fraction)?